plugins/arDominionB5Plugin/scss/_utilities.scss

```diff
@@ -0,0 +1,27 @@
+$utilities: () !default;
```

Comment:
It's to make sure our custom ones are added to the pre-existing/default ones.I thought it would be good to have just in case. Or is it unnecessary?

Reply:
The default is overwritten right after so it should be unnecessary. And, the way [`!default`](https://sass-lang.com/documentation/variables/#default-values) works in Sass, it won't make any effect since that variable is declared right before (inside `bootstrap/scss/utilities`).